api/serverV2.go

```diff
@@ -0,0 +1,63 @@
+package api
+
+import (
+	"github.com/iotexproject/iotex-core/action/protocol"
+	"github.com/iotexproject/iotex-core/actpool"
+	"github.com/iotexproject/iotex-core/blockchain"
+	"github.com/iotexproject/iotex-core/blockchain/blockdao"
+	"github.com/iotexproject/iotex-core/blockindex"
+	"github.com/iotexproject/iotex-core/blocksync"
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/state/factory"
+)
+
+type ServerV2 struct {
+	core       *CoreService
+	grpcServer *GrpcServer
+	httpServer *HttpServer
```

Comment:
web3Server

Reply:
i think remove `httpServer` in this PR?
in next PR, add it and the babel implementation